CONTRIBUTING.md

```diff
@@ -264,32 +273,30 @@ against two additional versions of libmongoc:
 
 #### Update sources in PECL package generation script
 
-If either libmongoc or libbson introduce a new source directory, that may also
+If a new version of a submodule introduces a new source directory, that may also
 require updating the glob patterns in the `bin/prep-release.php` script to
 ensure new source files will be included in any generated PECL package.
 
-#### Test and commit your changes
+#### Update SBOM file
 
-Verify that the upgrade was successful by ensuring that the driver can compile
-using both the bundled sources and system libraries for libmongoc and libbson,
-and by ensuring that the test suite passes. Once done, commit the changes to all
-of the above files/paths. For example:
+After updating dependencies, the SBOM file needs to be updated. There is a
+script to automate this process:
 
-```
-$ git commit -m "Bump libmongoc to 1.20.0" config.m4 config.w32 src/libmongoc src/LIBMONGOC_VERSION_CURRENT
+```shell
+./scripts/update-sbom.sh
 ```
 
-### Updating libmongocrypt
+This script will generate a purl file with our dependencies, then run the
```

Comment:
Do you want to clarify that the purl file is temporary, since `update-sbom.sh` removes it after invoking Docker?

Separate question: what is needed to "have access to artifactory"? I feel like this section would benefit from some links to internal resources.

Reply:
Added a "temporary" and removed the section about artifactory access - either it "just works" because I'm accessing it from our network or it is public - either way I don't imagine any external contributors running this kind of stuff on the regular anyways.